target/src/servo.c

```diff
@@ -0,0 +1,25 @@
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <rc_servo.h>
+#include "./include/servo.h"
+
+void servo_create(){
+
+	rc_servo_initialise();
```

Comment:
How come there is no need to check whether it was successful, like with the sensors?